tests/CMakeLists.txt

```diff
@@ -3,8 +3,17 @@
 ###############################################################################
 
 set(SRC_TEST_FILES
-  BasicTest.cpp
+        API/IEXTest.cpp
+        ../src/API/IEX.cpp
 )
 
-add_executable(UnitTests ${SRC_TEST_FILES})
-target_link_libraries(UnitTests CONAN_PKG::catch2)
+set(HDR_TEST_FILES
+        ../src/API/IEX.h)
+
+find_package(Catch2 REQUIRED)
```

Comment:
You don't need `find_package` as conan is providing the `CONAN_PKG::catch2` target for you already.